.github/workflows/image-scan-analysis.yml

```diff
@@ -1,61 +1,705 @@
-name: Security CI
+name: Image Security Scan
 
 on:
   push:
-    branches: [master, test, dev]
+    branches: [master, test]
   pull_request:
-    branches: [master, test, dev]
+    branches: [master, test]
   schedule:
-    - cron: '42 15 * * 5'
+    - cron: '30 5 14 * *' # Run this scan every sprint release cycle 14th of every month at 5:30 UTC
 
 jobs:
 
-  build_frontend:
+  nodejs-base-image:
```

Comment:
This looks like a number of repeated sections with just a few variables changing. This may be a good candidate for GHA matrixes.

https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#using-environment-variables-in-a-matrix